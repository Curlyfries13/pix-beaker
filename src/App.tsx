import React from 'react';
import logo from './logo.svg';
import './App.css';
import TextBox from './components/TextBox';
import CategorySelect from './components/CategorySelect';
import SearchButton from './components/SearchButton';

function App() {
  return (
    <div className="App">
      <TextBox />
      <CategorySelect />
      <SearchButton />
    </div>
  );
}

export default App;
