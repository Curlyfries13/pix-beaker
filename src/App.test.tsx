import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

it('renders learn react link', () => {
  const component: RenderResult = render(<App />);
  expect(component).toMatchSnapshot();
});
