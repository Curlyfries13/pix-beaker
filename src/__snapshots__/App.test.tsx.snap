// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <input
          aria-label=""
          type="text"
        />
        <select>
          <option
            value="fashion"
          >
            fashion
          </option>
          <option
            value="nature"
          >
            nature
          </option>
          <option
            value="backgrounds"
          >
            backgrounds
          </option>
          <option
            value="science"
          >
            science
          </option>
          <option
            value="education"
          >
            education
          </option>
          <option
            value="people"
          >
            people
          </option>
          <option
            value="feelings"
          >
            feelings
          </option>
          <option
            value="religion"
          >
            religion
          </option>
          <option
            value="health"
          >
            health
          </option>
          <option
            value="places"
          >
            places
          </option>
          <option
            value="animals"
          >
            animals
          </option>
          <option
            value="industry"
          >
            industry
          </option>
          <option
            value="food"
          >
            food
          </option>
          <option
            value="computer"
          >
            computer
          </option>
          <option
            value="sports"
          >
            sports
          </option>
          <option
            value="ansportation"
          >
            ansportation
          </option>
          <option
            value="travel"
          >
            travel
          </option>
          <option
            value="buildings"
          >
            buildings
          </option>
          <option
            value="business"
          >
            business
          </option>
          <option
            value="music"
          >
            music
          </option>
        </select>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <input
        aria-label=""
        type="text"
      />
      <select>
        <option
          value="fashion"
        >
          fashion
        </option>
        <option
          value="nature"
        >
          nature
        </option>
        <option
          value="backgrounds"
        >
          backgrounds
        </option>
        <option
          value="science"
        >
          science
        </option>
        <option
          value="education"
        >
          education
        </option>
        <option
          value="people"
        >
          people
        </option>
        <option
          value="feelings"
        >
          feelings
        </option>
        <option
          value="religion"
        >
          religion
        </option>
        <option
          value="health"
        >
          health
        </option>
        <option
          value="places"
        >
          places
        </option>
        <option
          value="animals"
        >
          animals
        </option>
        <option
          value="industry"
        >
          industry
        </option>
        <option
          value="food"
        >
          food
        </option>
        <option
          value="computer"
        >
          computer
        </option>
        <option
          value="sports"
        >
          sports
        </option>
        <option
          value="ansportation"
        >
          ansportation
        </option>
        <option
          value="travel"
        >
          travel
        </option>
        <option
          value="buildings"
        >
          buildings
        </option>
        <option
          value="business"
        >
          business
        </option>
        <option
          value="music"
        >
          music
        </option>
      </select>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
